docs/uninstall_and_downgrade.md

```diff
@@ -67,50 +63,32 @@ This procedure can be used to downgrade the active version of SAT.
    ###############################################################################
    Product Revision Information
    ###############################################################################
-   +--------------+-----------------+--------+--------------------+-----------------------+
-   | product_name | product_version | active | images             | image_recipes         |
-   +--------------+-----------------+--------+--------------------+-----------------------+
-   | sat          | 2.3.3           | True   | -                  | -                     |
-   | sat          | 2.2.10          | False  | -                  | -                     |
-   +--------------+-----------------+--------+--------------------+-----------------------+
+   +--------------+-----------------+--------------------+-----------------------+
+   | product_name | product_version | images             | image_recipes         |
+   +--------------+-----------------+--------------------+-----------------------+
+   | sat          | 2.3.3           | -                  | -                     |
+   | sat          | 2.2.10          | -                  | -                     |
+   +--------------+-----------------+--------------------+-----------------------+
    ```
 
-1. Use `prodmgr` to activate a different version of SAT.
+1. Use `prodmgr` to switch to a different version of SAT.
 
-   This command will do three things:
+   This command will do two things:
 
    - For all hosted-type package repositories associated with this version of SAT, set them as the sole member
-     of their corresponding group-type repository. For example, activating SAT version `2.2.10`
+     of their corresponding group-type repository. For example, switching to SAT version `2.2.10`
      sets the repository `sat-2.2.10-sle-15sp2` as the only member of the `sat-sle-15sp2` group.
-   - Set the version `2.2.10` as active within the product catalog, so that it appears active in the output of
-     `sat showrev`.
    - Ensure that the SAT CFS configuration content exists as a layer in all CFS configurations that are
      associated with NCNs with the role "Management" and subrole "Master" (for example, the CFS configuration
      `management-23.05`). Specifically, it will ensure that the layer refers to the version of SAT CFS
-     configuration content associated with the version of SAT being activated.
+     configuration content associated with the version of SAT to which you are switching.
 
    ```screen
    ncn-m001# prodmgr activate sat 2.2.10
    Repository sat-2.2.10-sle-15sp2 is now the default in sat-sle-15sp2.
-   Set sat-2.2.10 as active in product catalog.
```

Comment:
I'm realizing now that technically older version of the SAT product stream will still do this when they're "activated" with `prodmgr activate`. This is because they use the version of the `sat-install-utility` container image delivered with that version of the SAT product stream, and that older version will still set the "active" field in the product catalog. I don't care too much if you choose to still remove this line. With SAT 2.5 and beyond, this line will no longer be logged.

Reply:
@haasken-hpe, I think it'll be confusing to leave this line in, but I want the information to be accurate. I'll update the SAT version/information in the next commit to the following:

```screen
ncn-m001# prodmgr activate sat 2.5.15
Repository sat-2.5.15-sle-15sp4 is now the default in sat-sle-15sp4.
Updated CFS configurations: [management-23.04]
```